library/core/src/clone.rs

```diff
@@ -38,7 +38,25 @@
 
 mod uninit;
 
-/// A common trait for the ability to explicitly duplicate an object.
+/// A common trait that allows explicit creation of a "duplicate" value.
+///
+/// ## Important Note on "Duplication"
+///
```

Comment:
I don't think this heading adds much? At worst, it's a little confusing due to what else gets moved under it.
```suggestion
```